packages/migrate/src/__tests__/MigrateDev.test.ts

```diff
@@ -629,14 +629,14 @@ describe('sqlite', () => {
 
     await expect(result).rejects.toMatchInlineSnapshot(`
 
-                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                ⚠️ We found changes that cannot be executed:
+                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    ⚠️ We found changes that cannot be executed:
```

Comment:
Is this expected? I'm a bit puzzled

Reply:
Jest snapshots do that for a reason we don't completely understand.
You can ignore